### ISSUE TITLE
[Bug report] Missing external fluxes from split-transmon

### ISSUE THREAD

**Issue by QuantumWitness** [scqubits/scqubits#145]
### Documentation check
If the bug manifests in unexpected behavior (as opposed to a crash), confirm that you have consulted the 
[API documentation](https://scqubits.readthedocs.io/en/latest/api-doc/apidoc.html)

 - [ ] I have checked the API documentation.
 - [ ] I could not locate relevant information in the documentation or information is missing. 

### Describe the bug
When constructing a split-junction transmon from explicit circuit elements (not using the pre-made class), no external flux variables are created. Thus, flux biasing the transmon is impossible.

### Expected behavior
Expected one external flux variable to be able to bias the split-junction transmon. 

### To Reproduce
Here's the code I use to create the transmon:

```python
import scqubits as scqub

tmon_yaml = """# transmon circuit
branches:
- ["JJ", 1, 0, EJ=12,.01]
- ["JJ", 1, 0, EJ,0.01]
- ["C", 0, 1, EC=0.2]
"""

tmon = scqub.Circuit.from_yaml(tmon_yaml, from_file=False,
                               ext_basis="discretized")

tmon.sym_external_fluxes()
```

### OS and version used (please complete the following information):
 - OS: macOS 12.1
 - scqubits version 3.0.0
 - Python version 3.8

### Additional context
I tried this both with a floating and with a grounded transmon and received the same result. When I replace one junction with an inductor with $E_L = E_J$, I get the expected external flux variable, but not the expected eigenspectrum. I was hoping I could just define this using only junctions and a capacitor, to avoid making extra nodes, but perhaps that is just not possible?

EDIT: I guess I'm not sure if this is a bug, it's just not what I expected. What's the correct work-around?

Also do you want me to keep posting issues to get help figuring out scQubits or is there a discord/slack/somethingelse channel?

**Comment by jkochNU** [comment]
I can confirm that issue, and we are looking into a bug fix.

There's a _temporary_ workaround: switch the order of nodes in one of the two JJ branches:
```
tmon_yaml = """# transmon circuit
branches:
- ["JJ", 1, 0, EJ=12,.01]
- ["JJ", 0, 1, EJ,0.01]
- ["C", 0, 1, EC=0.2]
"""

**Comment by QuantumWitness** [comment]
I'm also noticing that the method `tmon.offset_charge_values()` returns an empty list, but I expected [0]. Maybe this has something to do with the weird eigenvalues I'm getting.

**Comment by pacosynthesis** [comment]
This issue has been addressed with https://github.com/scqubits/scqubits/commit/1dab4f11fdcd68ea87fdcb086de846d5355607ee and released with v3.0.2.
